common/utils/src/main/resources/error/error-classes.json

```
@@ -3854,6 +3854,11 @@
           "Catalog <catalogName> does not support <operation>."
         ]
       },
+      "COLLATION_SUPPORT_NOT_ENABLED" : {
```

Just feature name is enough:
```suggestion
      "COLLATION" : {
```